.github/actions/gradle-setup/action.yml

```diff
@@ -9,7 +9,7 @@ runs:
       with:
         repository: eclipse-dataspaceconnector/DataSpaceConnector
         path: DataSpaceConnector
-        ref: 5191d3e6dd9ac5d78264d05ae69edb4d297b606a
+        ref: 971b1a30598a2d7ddc622c73013d244a6e433027
```

Comment:
reminder to change commit when https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/1702 is merged